### PR TITLE
Tests2

### PR DESCRIPTION
**Pull request by stereobooster**
restored previously closed branch #88 

Everything updated and rebased

### CODE REVIEW

File: tests/main.test.js [hunk 1]
```diff
@@ -69,3 +69,68 @@ test('media queries print removed', async () => {
...
+  const result = ''
+  const { finalCss } = await runMinimalcss('fontface-removes')
+  expect(finalCss).toEqual(result)
+})
+
+test('leaves used @fontface', async () => {
+  const result =
+    "@font-face{font-family:'Lato';font-style:normal;font-weight:400;src:local('Lato Regular'),local('Lato-Regular'),url(https://fonts.gstatic.com/s/lato/v14/MDadn8DQ_3oT6kvnUq_2r_esZW2xOQ-xsNqO47m55DA.woff2) format('woff2');unicode-range:U+0000-00FF,U+0131,U+0152-0153,U+02C6,U+02DA,U+02DC,U+2000-206F,U+2074,U+20AC,U+2212,U+2215}.SomeSelector{font-family:'Lato'}"
+  const { finalCss } = await runMinimalcss('fontface-leaves')
+  expect(finalCss).toEqual(result)
```
Comment (peterbe) [inline tests/main.test.js]: Can't you do something like `expect(finalCss).toContain("@font-face{font-family:'Lato'")`?

File: tests/main.test.js [hunk 2]
```diff
@@ -69,3 +69,68 @@ test('media queries print removed', async () => {
...
+  const result =
+    '@media only screen and (min-device-width:414px) and (max-device-width:736px) and (-webkit-min-device-pixel-ratio:3){a{color:red}}@media only screen and (min-device-width:375px) and (max-device-width:812px) and (-webkit-min-device-pixel-ratio:3){a{color:green}}'
+  const { finalCss } = await runMinimalcss('media-queries')
+  expect(finalCss).toEqual(result)
+})
+
+test('evaluate DOM multiple times', async () => {
+  const result = '.SomeSelector{color:red}.OtherSelector{background:#000}'
+  const { finalCss } = await runMinimalcss('evaluate-dom-multiple-times')
+  expect(finalCss).toEqual(result)
```
Comment (peterbe) [inline tests/main.test.js]: Again, can this be simplified to `.toContain('.SomeSelector')` and `.toContain('.OtherSelector')`?

**Comment by stereobooster** [comment]
> Do you want to land this now?

I wan to land this. We can improve later

**Comment by stereobooster** [comment]
Will not update changelog for this. Will create PR your comments later

**Pull request merged**